pkg/server/translation.go

```diff
@@ -249,12 +249,11 @@ func (t *TranslateServiceServer) UpdateTranslation(
 		}
 
 		updatedTranslations = all
-
 	}
 
 	// Update messages for all translations
 	for i := range updatedTranslations {
-		err = t.repo.SaveTranslation(ctx, params.serviceID, &all[i])
+		err = t.repo.SaveTranslation(ctx, params.serviceID, &updatedTranslations[i])
```

Comment:
Having two instances of the same data will cause issues down the road. Is it possible to get rid off updatedTranslations and keep only one `all`?